docs/openapi_v1.0.0-alpha.yaml

```
@@ -299,16 +314,19 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
-  /cancellations:
+  /suppliers/{supplierId}/bookings/{uuid}/cancel:
     post:
       tags:
-        - Cancellations
+        - Bookings
```

This should still be tagged as a Cancellation. The tag doesn't refer to the path, rather it's a way to group endpoints based on the type of operation you're trying to do. In this case, the endpoint is only applicable to a Cancellation.

```suggestion
        - Cancellations
```